community/teams/documentation.tt

```diff
@@ -15,16 +15,47 @@
 <section class="governance-team">
 
   <aside>
-    <h2>Members</h2>
 
+    <h2>Leader</h2>
     <ul>
       <li>
         Valentin Gagarin (<a href="https://discourse.nixos.org/u/fricklerhandwerk">@fricklerhandwerk</a>)<br/>
-        Nix documentarian, sponsored by <a href="https://antithesis.com/">Antithesis</a> via <a href="https://tweag.io">Tweag</a>
```

Comment:
I think we should keep information on affiliations, as that's easy to find as opposed to PR annotations, and I wouldn't know a more suitable and easier to maintain places.

Reply:
Sounds good too, I suggested to remove it originally to keep it simple and the same for everybody, but I guess people can always PR to add their affiliations later if desired.

Reply:
When you say “affiliations” are you referring to employers or sponsors? I have an employer (flox) but they aren’t _sponsoring_ my work. Should they be listed?

Reply:
I think this should be up to you :)

Reply:
@henrik-ch Let's keep the existing affiliations then, can you update the PR as such?